### PR TITLE
[TG-1] Purposely break test

### PR DESCRIPTION
**Pull request by gblock0**

### CODE REVIEW

File: TestGit.Tests/Tests.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ public void Test1()
         {
             var args = new[] {"test2", "test3"};
             var firstArg = TestGit.ParseOutFirstArg(args);
-            Assert.AreEqual(firstArg, "test2");
+            Assert.AreEqual(firstArg, "test3");
```
Comment (ghost) [inline TestGit.Tests/Tests.cs]: I think this should be `test2`?
```suggestion
            Assert.AreEqual(firstArg, "test2");
```

**Review by ghost — CHANGES_REQUESTED** [review]
This needs to be fixed (also the test is broken)

File: TestGit.Tests/Tests.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@ public void Test1()
         {
             var args = new[] {"test2", "test3"};
             var firstArg = TestGit.ParseOutFirstArg(args);
-            Assert.AreEqual(firstArg, "test2");
```
Comment (gblock0) [inline TestGit.Tests/Tests.cs]: This is good otherwise...

**Comment by gblock0** [comment]
This will fix [TG-1]

[TG-1]: https://testgit.atlassian.net/browse/TG-1

**Comment by gblock0** [comment]
^ That revert was made with 2 clicks in the Github Desktop UI :D

**Comment by gblock0** [comment]
TG-1 Test

File: TestGit/Program.cs [hunk 3]
```diff
@@ -12,6 +12,7 @@ public static void Main(string[] args)
         public static string ParseOutFirstArg(string[] args)
         {
             // change some stuff
+            // add comment
             return args[0];
```
Comment (techgerm) [inline TestGit/Program.cs:16]: Something is up here!

File: TestGit.Tests/Tests.cs [hunk 4]
```diff
@@ -9,9 +9,9 @@ public class Tests
         [Test]
         public void Test1()
         {
-            var args = new[] {"test2", "test3"};
+            var args = new[] {"test2", "test3", "test4"};
```
Comment (techgerm) [inline TestGit.Tests/Tests.cs:12]: Maybe test 5?

**Review by techgerm — CHANGES_REQUESTED** [review]
Needs more work!

**Review by techgerm — APPROVED** [review]

**Pull request closed without merging**